data/src/main/java/org/oppia/android/data/backends/gae/NetworkConfigModule.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.data.backends.gae
+
+import dagger.Module
+import dagger.Provides
+
+/** Provides Network constants specifically related to network only. .*/
+@Module
+class NetworkConfigModule {
```

Comment:
Suggest: ``NetworkConfigProdModule`` since you presumably will want to introduce a test module.

Reply:
Done.
didn't change it in the instrumentation module, `EndToEndTestNetworkConfigModule` seemed fine to me